clients/venice-admin-tool/src/main/java/com/linkedin/venice/Command.java

```diff
@@ -437,6 +437,10 @@ public enum Command {
   EXECUTE_DATA_RECOVERY(
       "execute-data-recovery", "Execute data recovery for a group of stores",
       new Arg[] { RECOVERY_COMMAND, STORES, SOURCE_FABRIC }, new Arg[] { EXTRA_COMMAND_ARGS, DEBUG, NON_INTERACTIVE }
+  ),
+  ESTIMATE_DATA_RECOVERY_TIME(
+      "estimate-data-recovery-time", "Estimates the time it would take to execute data recovery for a group of stores.",
+      new Arg[] { URL, STORES, CLUSTER }
```

Comment:
I feel like we may want to relax the constraint of having CLUSTER as required arguments. @gaojieliu once mentioned that it could be possible that the input stores are not always from the same cluster and current we don't have such constraint for other two admin-tool commands, so it's better to relax it. Alternatively, we can build our own `ControllerClient` internally for different clusters, if cluster is a must, or we can simply use a dummy cluster, if cluster is not required. See an example here: https://github.com/linkedin/venice/pull/301